hack/perf/bz-2162111.sh

```diff
@@ -8,13 +8,20 @@ file_b=$(mktemp -p $tmp --suffix '.file_b')
 dd if=/dev/zero of=$file_a bs=1024 count=1024 status=none
 dd if=/dev/zero of=$file_b bs=1024 count=1024 status=none
 
-# The create command
 volume_name="bz-2162111"
 container_name="bz-2162111"
+network_name="bz-2162111"
+
+$ENGINE_A network rm -f $network_name >> /dev/null || true
+$ENGINE_B network rm -f $network_name >> /dev/null || true
+$ENGINE_A network create $network_name >> /dev/null
+$ENGINE_B network create $network_name >> /dev/null
```

Comment:
Any reason to not use random names? That would prevent all name conflicts.

Also I am missing where you remove the network after the test is done?

Reply:
> Any reason to not use random names? That would prevent all name conflicts.

Easier debugging to know where it came from.

> Also I am missing where you remove the network after the test is done?

I didn't intend to since it'll get remove on the next invocation.  The other scripts may also leave containers etc. in storage.

Reply:
I'm with @Luap99 on this one. It is polite to clean up after oneself.

Reply:
I don't like leaking test resources for no reason.
I expected all test to cleanup what they created, I understand keeping images because pulling is expensive but the rest not so much.

Reply:
Please note the `./hack` , it's not in `./test`.  I am not arguing against any of these points but ask to apply different standards to these hacked scripts.

I'll repush with cleaning up nonetheless :+1:

Reply:
Updated.  Added a commit on top to also clean up after the other benchmarks.